x5.md

```diff
@@ -8,7 +8,118 @@ created: 2019-04-18
 updated:
 ---
 
-# Introduction
+# Trust Establishment Protocol Specification Draft
+## Inspiration
+- https://datprotocol.github.io/how-dat-works/
```

Comment:
This seems unnecessary

Reply:
there were a lot of draft notes in that doc.. like the other, it wasn't cleaned up